tests/system_tests/test_blueapi_system.py

```diff
@@ -122,11 +121,15 @@ def test_cannot_access_endpoints(
     for get_method in blueapi_client_get_methods:
         with pytest.raises(BlueskyRemoteControlError) as exception:
             getattr(client_without_auth, get_method)()
-        assert str(exception) == "<Response [401]>"
+        assert str(exception.value) in "<Response [401]>"
```

Comment:
Shouldn't this be the other way around? 
And was there a reason to not use the `raises(match=`?

Reply:
I have added match for this now